stable/jenkins/templates/jenkins-master-deployment.yaml

```diff
@@ -105,6 +105,9 @@ spec:
           resources:
 {{ toYaml .Values.Master.resources | indent 12 }}
           volumeMounts:
+            -
```

Comment:
while we are visiting this area, can we make all the volumeMounts look better? :)
as in, instead of
```
-
  mountPath:
  name:
```
make it:
```
- mountPath:
  name:
```

unless there is a hidden reason for this which i am missing...